controllers/flowcollector_controller.go

```diff
@@ -5,6 +5,7 @@ import (
 	"net"
 	"strings"
 
+	//TODO replace ViaQ repo by grafana/loki/operator one when released
```

Comment:
The operator is in https://github.com/grafana/loki/ ...  maybe this remark is not needed?

Reply:
I'm looking for `grafana/loki/operator` not `loki` itself and it's [not yet released](https://pkg.go.dev/github.com/grafana/loki/operator)

Reply:
ok